### DISCUSSION TITLE
Flows: Run Script Renaming

### DISCUSSION THREAD

**Discussion by MentalGear** [directus/directus#16682] [Feature Requests]
### Proposal
Rename the 'data' variable that is returned / prefilled as default by Run Script with something more precise and meaningful like "flowChain".

### Why
The term "Data" is too generic. As a user connecting the script operation from another flow, it's the most obvious notion by context to think that "data" is the output of the last operation. However, "data" is the full "chain" and all it's props. This is also hard to figure out since errors in Run Script fail the whole flow silently (an error log for failed flows would be great) and no log is available. A more precise name for what the data actually is would do very much in preventing this error and bring clarity about what we are using.

```
module.exports = async function( fullFlowChain ) {
	// Do something...
	return {};
}
```

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
